src/c_kzg_4844.h

```diff
@@ -29,6 +29,14 @@
 
 #include "blst.h"
 
+// Allow a library built from this code to be used from C++
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+#define BYTES_PER_COMMITMENT 48
+#define BYTES_PER_PROOF 48
+#define BYTES_PER_FIELD 32
```

Comment:
this must be an old version of the file. the latest one defines `BYTES_PER_FIELD_ELEMENT` - maybe you can use that?